.ci/tav.json

```diff
@@ -1,38 +1,38 @@
 {
-  "versions": [ "20", "19", "18", "16", "14" ],
+  "versions": [ 20, 19, 18, 16, 14 ],
   "modules": [
-    "@apollo/server",
-    "@aws-sdk/client-s3",
-    "@elastic/elasticsearch",
-    "@hapi/hapi",
-    "@opentelemetry/api",
-    "@opentelemetry/sdk-metrics",
-    "apollo-server-express",
-    "aws-sdk",
-    "cassandra-driver",
-    "elasticsearch",
-    "express",
-    "express-queue",
-    "fastify",
-    "finalhandler",
-    "generic-pool",
-    "graphql",
-    "ioredis",
-    "knex",
-    "memcached",
-    "mongodb",
-    "mongodb-core",
-    "mysql",
-    "mysql2",
-    "next",
-    "pg",
-    "redis",
-    "restify",
-    "tedious",
-    "undici",
-    "ws",
-    "@koa/router,koa-router",
-    "handlebars,pug",
-    "bluebird,got"
+    {"name": "@apollo/server", "minVersion": 18 },
```

Comment:
Note for reviewer: to run less tests we add `minVersion` to 18. This shall be updated with the proper versions if the PR is okay to merge